frontend/src/routes/Governance/discovered/DiscoveredPolicies.tsx

```diff
@@ -0,0 +1,205 @@
+        search: 'violations',
+        id: 'violations',
+        exportContent: (item: DiscoverdPolicyTableItem) => {
+          const { noncompliant, compliant, pending, unknown } = policyViolationSummary(item.policies)
+          return `${t('no violations: {{count}} cluster', { count: compliant })}, ${t('violations: {{count}} cluster', { count: noncompliant })}, ${t('pending: {{count}} cluster', { count: pending })}, ${t('unknown: {{count}} cluster', { count: unknown })}`
+        },
+      },
+    ],
+    [t]
+  )
+
+  const filters = useMemo<ITableFilter<DiscoverdPolicyTableItem>[]>(
+    () => [
+      {
+        id: 'violations',
+        label: 'Cluster violations',
+        options: [
+          {
+            label: t('No violations'),
+            value: 'no-violations',
+          },
+          {
+            label: t('Violations'),
+            value: 'violations',
+          },
+        ],
+        tableFilterFn: (selectedValues, item) => {
+          const { noncompliant, compliant } = policyViolationSummary(item.policies)
+
+          if (selectedValues.includes('no-violations') && noncompliant == 0) {
+            return true
+          } else if (selectedValues.includes('violations') && compliant > 0) return true
+          return false
+        },
+      },
+      {
+        id: 'kind',
+        label: t('Kind'),
+        options: [
+          { label: t('Configuration policy'), value: 'ConfigurationPolicy' },
+          { label: t('Operator policy'), value: 'OperatorPolicy' },
+          { label: t('Certificate policy'), value: 'CertificatePolicy' },
+        ],
+        tableFilterFn: (selectedValues, item) => {
+          return selectedValues.includes(item.kind)
+        },
+      },
+      {
+        id: 'responseAction',
+        label: t('Response action'),
+        options: [
+          { label: t('Inform'), value: 'inform' },
+          { label: t('Enforce'), value: 'enforce' },
+          { label: t('InformOnly'), value: 'informOnly' },
+          { label: t('Enforce/Inform'), value: 'enforce/inform' },
+        ],
+        tableFilterFn: (selectedValues, item) => {
+          return selectedValues.includes(item.responseAction)
+        },
+      },
+    ],
+    [t]
+  )
+
+  if (isFetching) {
+    return (
+      <PageSection>
+        <EmptyState>
+          <EmptyStateIcon variant="container" component={Spinner} />
+          <Title size="lg" headingLevel="h4">
+            Loading
+          </Title>
+        </EmptyState>
+      </PageSection>
+    )
+  }
+  /* t('preview.discoveredPolicies') */
+  return (
+    <PageSection>
+      <TechPreviewAlert i18nKey="preview.discoveredPolicies" docHref={DOC_LINKS.POLICIES_OVERVIEW} />
+      <AcmTable<DiscoverdPolicyTableItem>
+        id="discoveredPolicyTable"
+        columns={discoveredPoliciesCols}
+        keyFn={(item) => item.id}
+        items={data}
+        emptyState={<AcmEmptyState title={t('No polices found')} message={t('There are no policies')} />}
```

Comment:
@mprahl  @dockerymick  Is this enough? maybe is there a better sentence?

Reply:
title: "You don't have any policies"
message: "Click the Create policy  button to create your resource. View the documentation for more information."

Reply:
The user cannot create Cert/Config/Operator policy directly and there is no create button for these policies

Reply:
ok, thanks for confirming. What can the user do then about not having policies?

Reply:
The user can create Cert/Config/Operator policies through `kind: policy`, Create `Policy` with object template(Cert/Config/Operator)

Reply:
or ArgoCD